app/src/main/java/org/redcross/openmapkit/MapActivity.java

```diff
@@ -1014,6 +1017,9 @@ private boolean isGPSEnabled() {
     }
 
     private void showProgressDialog() {
+        if(isUserLocationEnabled() == false) {
```

Comment:
Is there a specific reason why I should use '!' instead of false? I find using false better, especially when it comes to skimming through the code. Very hard to miss a false